### DISCUSSION TITLE
Is it possible to train and make models like Hassan blend with this dreambooth?

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#1491] [Q&A] [answered]
I would love to make something like Hassan blend. how to do that on colab? Thanks!

**Comment by TheLastBen** [comment, marked as answer]
That model is trained using dreambooth

**Reply by gilroff** [reply]
how to do that with the limit of 20 instance images?

**Reply by TheLastBen** [reply]
more images = more diversity = need more training steps

**Reply by gilroff** [reply]
then, do i need to lower learning rate to do this?
can you explain how to do all this?

**Reply by TheLastBen** [reply]
search in youtube "dreambooth tutorial" you will find plenty of examples using this colab.

**Reply by gilroff** [reply]
ok thanks!
